README.md

```diff
@@ -98,15 +98,131 @@ This release extends support for RHEL v8.5.
+| gfx803        | Polaris    | Polaris 30      | AMD Radeon™ RX 590                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XL      | AMD Radeon™ RX Vega 56                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XL      | AMD Radeon™ PRO Vega 56                           |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ Instinct MI25                         |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ RX Vega 64                            |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XT      | AMD Radeon™ RX Vega 64 Liquid                     |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XTX AIR | AMD Radeon™ Vega Frontier Edition                 |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx900        | Vega       | Vega 10 XTX LCS | AMD Radeon™ Vega Frontier Edition (Liquid Cooled) |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx906:xnack- | Radeon VII | Vega 20 GL      | AMD Radeon™ Instinct MI50                         |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx906:xnack- | Radeon VII | Vega 20 GL      | AMD Radeon™ Instinct MI60                         |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx906:xnack- | Radeon VII | Vega 20 XT      | AMD Radeon™ VII                                   |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx906:xnack- | Radeon VII | Vega 20 XT      | AMD Radeon™ PRO VII                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ∅    |
+| gfx908:xnack- | MI100      | MI100 XL        | AMD Instinct MI100                                |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
+| gfx90a:xnack+ | MI210      | Aldebaran       | TBA                                               |  ✅   |    ✅    |   ✅    |    ✅    |    ✅    |     ✅     |     ✅     |     ✅     |     ✅      |    ✅    |
```

Comment:
gfx90a is the MI200 family of devices. The chips are named Aldebaran in the amdgpu driver, The product names are mentioned above: AMD Instinct(tm) MI210, MI250, and MI250X.